v21.1/inverted-indexes.md

```diff
@@ -143,9 +143,23 @@ If you require comparisons using [`<`](functions-and-operators.html#supported-op
 - "is contained by": [`<@`](functions-and-operators.html#supported-operations)
 - "contains": [`@>`](functions-and-operators.html#supported-operations)
 
+## Partial inverted indexes
+
+<span class="version-tag">New in v21.1:</span> You can create a [partial](partial-indexes.html) inverted index, an inverted index on a subset of `JSON`, `ARRAY`, or geospatial container column data. Just like partial indexes that use non-container datatypes, create a partial inverted index by including a clause that evaluates to true on a boolean predicate, like a `WHERE` clause.
+
+{% include copy-clipboard.html %}
+~~~ sql
+CREATE TABLE test (
+  id INT,
+  data JSONB,
+  INVERTED INDEX idx_data(data) WHERE id > 10
+);
+~~~
+
 ## Known limitations
 
-CockroachDB does not support partitioning inverted indexes. For details, see [tracking issue](https://github.com/cockroachdb/cockroach/issues/43643).
+- CockroachDB does not support partitioning inverted indexes. For details, see [tracking issue](https://github.com/cockroachdb/cockroach/issues/43643).
```

Comment:
This is supported in 21.1. The inverted index has to be multi-column so that there is a non-container column to partition by.